packages/fxa-payments-server/src/components/PaymentConfirmation/index.tsx

```diff
@@ -71,57 +69,63 @@ export const PaymentConfirmation = ({
   const downloadUrl: URL = new URL(productUrl);
   downloadUrl.searchParams.append('email', email);
 
+  // TW classes
+  const subheadingClasses = 'text-grey-400 max-w-sm';
+  const h2classes = 'text-xl font-normal mx-0 mt-6 mb-3';
+  const h3classes = 'text-sm font-semibold';
+  const bottomRowClasses = 'flex justify-between items-center text-grey-400';
+
   return (
     <>
       <SubscriptionTitle screenType="success" />
       <section
-        className={`container card payment-confirmation ${className}`}
+        className="container payment-confirmation"
         data-testid="payment-confirmation"
       >
-        <header>
+        <header className="flex flex-col justify-center items-center row-divider-grey-200 text-center pb-8 mt-5 desktop:mt-2">
           {accountExists ? (
             <>
               <img
-                className="circled-check"
+                className="max-h-12"
                 src={circledCheckbox}
                 alt="circled checkbox"
               />
               <Localized id="payment-confirmation-thanks-heading">
-                <h2>Thank you!</h2>
+                <h2 className={h2classes}>Thank you!</h2>
               </Localized>
               <Localized
                 id="payment-confirmation-thanks-subheading"
                 vars={{ email, product_name }}
               >
-                <p>{`A confirmation email has been sent to ${email} with details on how to get started with ${product_name}.`}</p>
+                <p
+                  className={subheadingClasses}
+                >{`A confirmation email has been sent to ${email} with details on how to get started with ${product_name}.`}</p>
               </Localized>
             </>
           ) : (
             <>
               <img src={checkmarkIcon} alt="checkmark icon" />
-              <img
-                className="email-sent-icon"
-                src={emailSentIcon}
-                alt="email sent icon"
-              />
+              <img src={emailSentIcon} alt="email sent icon" />
               <Localized id="payment-confirmation-thanks-heading-account-exists">
-                <h2>Thanks, now check your email!</h2>
+                <h2 className={h2classes}>Thanks, now check your email!</h2>
               </Localized>
               <Localized
                 id="payment-confirmation-thanks-subheading-account-exists"
                 vars={{ email }}
               >
-                <p>{`You'll receive an email at ${email} with instructions for setting up your account as well as  your payment details.`}</p>
+                <p
+                  className={subheadingClasses}
+                >{`You'll receive an email at ${email} with instructions for setting up your account as well as  your payment details.`}</p>
```

Comment:
Nit: I don't think it matters bc it's correct in `main.ftl` but the apostrophe in "You'll" should be ’.

Reply:
Updated to maintain consistency 🙂 !